docs/ops.html

```diff
@@ -1447,12 +1447,32 @@ <h4 class="anchor-heading"><a id="remote_jmx" class="anchor-link"></a><a href="#
         <td>kafka.network:type=RequestMetrics,name=RequestsPerSec,request={Produce|FetchConsumer|FetchFollower},version=([0-9]+)</td>
         <td></td>
       </tr>
+      <tr>
+        <td>Produce request rate</td>
+        <td>kafka.server:type=BrokerTopicMetrics,name=TotalProduceRequestsPerSec</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>Fetch request rate</td>
+        <td>kafka.server:type=BrokerTopicMetrics,name=TotalFetchRequestsPerSec</td>
+        <td></td>
+      </tr>
       <tr>
         <td>Error rate</td>
         <td>kafka.network:type=RequestMetrics,name=ErrorsPerSec,request=([-.\w]+),error=([-.\w]+)</td>
         <td>Number of errors in responses counted per-request-type, per-error-code. If a response contains
             multiple errors, all are counted. error=NONE indicates successful responses.</td>
       </tr>
+      <tr>
+        <td>Failed produce request rate</td>
+        <td>kafka.server:type=BrokerTopicMetrics,name=FailedProduceRequestsPerSec</td>
+        <td></td>
```

Comment:
seems like we're missing an explanation for what these metrics measure.

Reply:
Yes let's try to include descriptions as above we recommend to graph and alert on these metrics